protected/views/patient/crud/_form.php

```diff
@@ -475,7 +475,7 @@
         [
             'type'=>'POST',
             'error'=>'js:function(){
-      alert("error");
+      alert("Fill the mandatory fields");
```

Comment:
A "please" is always nice :) ("Please fill in the fields marked as mandatory" or something)
However you might want to show the errors that the controller generates instead (in case there is something other than mandatoriness issues) or just say "please correct the highlighted errors"
Also I wouldn't use alert() for displaying a dialog to the user, there is a nice openeyes specific dialog that would be be more fitting.